contrib/bundle/build

```diff
@@ -36,7 +36,7 @@ FILES_ETC=(
     "$GIT_ROOT/contrib/bundle/10-crun.conf"
 )
 FILES_CONTRIB=(
-    "$GIT_ROOT/contrib/cni/10-crio-bridge.conf"
+    "$GIT_ROOT/contrib/cni/10-crio-bridge.conflist"
```

Comment:
why the rename to conflist?

Reply:
That's my question too.  If the name change is valid, the rest LGTM.

Reply:
My experiments show, that when the configuration file has `plugins:` then it must have also .conflist extension.

Reply:
See also https://github.com/cri-o/cri-o/pull/6289#issuecomment-1274156122.

Reply:
@haircommander As @dilyanpalauzov mentioned the configurations file now has `plugins` so I have to change the extensions of the filename from `.conf` to `.conflist`.

Hence, this change.